### DISCUSSION TITLE
Bulma with Tailwind

### DISCUSSION THREAD

**Discussion by AsiwajuDev** [tailwindlabs/tailwindcss#2117] [Help]
I'm working on a NextJs app converting a Bulma template into components. I've setup Bulma and trying to add tailwind so I could eliminate the stress of manually adding some CSS classes and make design uniform across all pages. I've tried the normal set-up used in React but I got a posscss config object error

**Comment by estevanmaito** [comment]
Could you share the error?

**Reply by AsiwajuDev** [reply]
Shutdown my system already....can you just recommend a material to use or you could just list the steps for me.

**Reply by Apostolique** [reply]
@AsiwajuDev Did you already looks at: <https://blog.tailwindcss.com/building-the-tailwind-blog> and <https://github.com/tailwindlabs/blog.tailwindcss.com>?

**Reply by AsiwajuDev** [reply]
This didn't really help in anyway
